app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -225,6 +225,15 @@ def content_override
       respond_for_show(:resource => @activation_key)
     end
 
+    api :GET, "/activation_keys/:id/product_content", N_("Show content available for an activation key")
```

Comment:
Should this API be deprecated for removal in say ... 4.3?

Reply:
Which API replaces it?

Reply:
> Should this API be deprecated for removal in say ... 4.3?

May be .. But I would consider leaving it as it is since the rest of the activation key actions use the `/activation_keys/:id/<action>` url

> Which API replaces it?
`/repository_sets` now accepts an activation key id or a  host id to give the same content.

Reply:
Ahh, I see, that was part of your #9219.

We *could* decide in FAM which action to execute, depending on availability, but not having to do it is easier for sure ;)